Presentation.Web/app/services/ui-customization/ui-customization-service.ts

```diff
@@ -30,6 +30,27 @@
         visitModule(node: Models.UICustomization.CustomizedModuleUI) { /* We don't care much for modules - we care about the nodes */ }
     }
 
+    export class CountNodeStatesFromUiCustomizationTreeVisitor implements Models.UICustomization.IUICustomizationTreeVisitor {
+        private _nodesByStateCounter: number = 0;
+        private _state: boolean;
```

Comment:
since we compare with "available" then the right name for this class would be:
`CountNodesByAvailableStateUiCustomizationTreeVisitor`
and the right name for that field would be
`_matchAvailableState`